java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java

```diff
@@ -31,15 +29,6 @@
         security = {
                 @SecurityRequirement(name = "req 1", scopes = {"a", "b"}),
                 @SecurityRequirement(name = "req 2", scopes = {"b", "c"})
-        },
-        servers = {
-                @Server(
-                        description = "server 1",
-                        url = "http://foo",
-                        variables = {
-                                @ServerVariable(name = "var1", description = "var 1", defaultValue = "1", allowableValues = {"1", "2"}),
-                                @ServerVariable(name = "var2", description = "var 2", defaultValue = "1", allowableValues = {"1", "2"})
-                        })
         }
```

Comment:
why are these removed?

Reply:
Recovered, and added a new Server for testing .